sdk/storage/azure-storage-blob/README.md

```diff
@@ -105,6 +108,30 @@ Blob Storage is designed for:
 - Storing data for backup and restore, disaster recovery, and archiving
 - Storing data for analysis by an on-premises or Azure-hosted service
 
+### URL format
+Blobs are addressable using the following URL format:
+The following URL addresses a blob:
+https://myaccount.blob.core.windows.net/mycontainer/myblob
+
+#### Resource URI Syntax
+For the storage account, the base URI for blob operations includes the name of the account only:
+
+```$xslt
```

Comment:
Does that refer to https://en.wikipedia.org/wiki/XSLT ? I wonder why such choice here.

Reply:
tbh I copied it from the queue README

Reply:
This is what it ends up generating as (take a look at the url part of the README) https://github.com/Azure/azure-sdk-for-java/tree/master/sdk/storage/azure-storage-queue

Reply:
It doesn't seem to bring any special formatting when I see preview of this page. Why don't we drop it?
It brings back some bad memories from the past.

Reply:
sure, I'll remove it

Reply:
removed!